docs/hub/spaces-sdks-docker-giskard.md

```diff
@@ -0,0 +1,125 @@
+- Compare models to decide which model to promote
+
+- Collect business feedback of your model results
+
+- Share your results with your colleagues for alignment
+
+- Store all your QA objects (tests, data slices, evaluation criteria, etc.) in one place to work more efficiently
+
+Visit [Giskard's documentation](https://docs.giskard.ai/) and [Quickstart Guides](https://docs.giskard.ai/en/latest/getting_started/quickstart/index.html) 
+to learn how to use the full range of tools provided by Giskard.
+
+In the next sections, you'll learn to deploy your own Giskard AI Model Quality Testing app and use it right from 
+HuggingFace Spaces. This Giskard app is a **self-contained application completely hosted on Spaces using Docker**.
+
+## Deploy Giskard on Spaces
+
+You can deploy Giskard on Spaces with just a few clicks:
+
+<a  href="https://huggingface.co/new-space?template=giskardai%2Fgiskard">
+    <img src="https://huggingface.co/datasets/huggingface/badges/resolve/main/deploy-to-spaces-lg.svg" />
+</a>
+
+
+<Tip>
+IMPORTANT NOTE ABOUT DATA PERSISTENCE:
+You can use the Giskard Quickstart Space as is for initial exploration and experimentation. For **longer use in 
+small-scale projects, activate the paid persistent storage option**. This prevents data loss during Space restarts which 
+occur every 24 hours.
+</Tip>
+
+You need to define the **Owner** (your personal account or an organization), a **Space name**, and the **Visibility**. 
+If you don’t want to publicly share your models and quality tests, set your Space to **Private**.
+
+<div class="flex justify-center">
+<img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/spaces-giskard-new-space.png"/>
+</div>
+
+Once you have created the Space, you'll see the `Building` status and once it becomes `Running` your space is ready to go. 
+If you don't see a change in the screen, refresh the page.
+
+## Request a free license
+
+Once your Giskard Space is up and running, you'll need to request a free license to start using the app. 
+You will then automatically receive an email with the license file. 
+
+<div class="flex justify-center">
+<img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/spaces-giskard-free-license.png"/>
+</div>
+
+## Create a new Giskard project
+
+Once inside the app, start by creating a new project from the welcome screen.
+
+<div class="flex justify-center">
+<img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/spaces-giskard-create-project.png"/>
+</div>
+
+## Generate a Giskard Space token
+
+The Giskard Space token is used to establish communication between the environment where your AI models are running and 
+the Giskard HuggingFace Space. If you've set the **Visibility** of your Space to **Private**, you will need to provide a HuggingFace 
+access token to generate the Giskard Space Token and establish a communication. To do so, follow the instructions 
```

Comment:
Let's build the link between the semantic `Giskard Space Token` and the `HF-TOKEN` used below.

HF users may be confused by `HF-TOKEN` and their [HF User Access token](https://huggingface.co/docs/hub/main/en/security-tokens).

Reply:
Ah ok, so what should we name each of them?

Reply:
I think either we call it SPACE-TOKEN, or we can explicitly say HF_TOKEN is the Giskard Space Token